src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -462,6 +468,8 @@ class EmojiPickerMenu extends Component {
                     this.setState({highlightedIndex: -1});
                 }}
                 emoji={emojiCode}
+                onFocus={() => this.setState({highlightedIndex: index})}
+                isFocused={index === this.state.highlightedIndex && this.state.isUsingKeyboardMovement}
```

Comment:
nit: just for readability, put this variable somewhere above
`const  isEmojiFocused = index === this.state.highlightedIndex && this.state.isUsingKeyboardMovement`

```suggestion
                isFocused={isEmojiFocused}
```